apis/core/bootstrap/shapes/csv-source.ts

```diff
@@ -34,8 +44,19 @@ ${shape('csv-source/update')} {
       ${sh.datatype} ${xsd.string} ;
       ${sh.minCount} 1 ;
       ${sh.maxCount} 1 ;
+      ${sh.minLength} 1 ;
       ${sh.order} 20 ;
     ] ;
+    ${sh.property} [
+      ${sh.name} "Header rows" ;
+      ${sh.path} ${csvw.headerRowCount} ;
+      ${sh.datatype} ${xsd.integer} ;
+      ${sh.minCount} 1 ;
+      ${sh.maxCount} 1 ;
+      ${sh.default} 1 ;
+      ${sh.minInclusive} 0 ;
+      ${sh.order} 30 ;
+    ] ;
```

Comment:
This get's set to `1` automatically by the sniffer. Might be necessary to tweak in a rare case